xAPI.md

```diff
@@ -438,6 +439,16 @@ __Registration__: An instance of a learner experiencing a particular Activity.
 __Representational State Transfer (REST)__: An architecture for designing networked web Services.
 It relies on HTTP methods and uses current web best practices.
 
+<a name="def-reporting-system" />
+
+__Reporting System__: A system or tool used to report on and analyse xAPI data and display this to the user. 
```

Comment:
The class of tool that we're talking about here is a tool which renders data to end users looking at the tracking data. Sounds like that doesn't match up with what you think about when you read "reporting system". Perhaps we need a different term? Suggestions?

The other option is to ditch the definitions and go for something more vague wherever it occurs like 'tools that render data' but I'd rather avoid that if possible.